easybuild/easyconfigs/m/malt/malt-0.4.1.eb

```diff
@@ -0,0 +1,40 @@
+# This is for the custom build of Malt 
+# Malt is downloaded from here:
+# http://ab.inf.uni-tuebingen.de/data/software/malt/download/welcome.html
+# The MALT_unix_0_4_1.sh is executed and the binaries are installed in
+# malt/bin
+# The malt directory is bundled into a tarball which will be installed
+# This is a bit of a hack!
+# The homepage is here:
+# http://ab.inf.uni-tuebingen.de/software/malt/
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+easyblock = 'PackedBinary'
+
+name = 'malt'
+version = '0.4.1'
+
+homepage = 'http://ab.inf.uni-tuebingen.de/software/malt/'
+description = """MALT (MEGAN alignment tool) is an extension of MEGAN (metagenome analyzer).
+
+MALT performs alignment of metagenomic reads against a database of reference sequences 
+(such as NR, GenBank or Silva) and produces a MEGAN RMA file as output. The software is 
+currently under development. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['malt-%(version)s.tar.gz']
```

Comment:
@sassy-crick Where did you download this from? I don't see it at http://ab.inf.uni-tuebingen.de/data/software/malt/download/welcome.html ?

Reply:
@boegel I was downloading the installation script from that URL, installed it in my space and tarballed the folder. As I said in the comment, that is a hack. Let me know how to do that better and I am happy to do this. As I am still having problems with the latest GAMESS to get that configured I did not want to create another issue for me, if you see what I mean. That is the reason for my comment on top of the file. :-)

Reply:
Sounds like a case for the `CmdCp` easyblock to me...

Reply:
There is actually more to it. The author does not do a proper version, so everything is 'master' which makes it impossible to rebuild anything. Unless somebody has a good idea here, I would suggest to close that pull request.